src/main/java/org/sciborgs1155/robot/intake/IntakeConstants.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.sciborgs1155.robot.intake;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile.Constraints;
+import org.sciborgs1155.lib.constants.ArmFFConstants;
+import org.sciborgs1155.lib.constants.PIDConstants;
+import org.sciborgs1155.robot.Constants;
+
+/** Add your docs here. */
```

Comment:
🤨
(you don't need to write a docstring if you don't want to, but the *add docs here* is cursed)